app/src/main/res/values/strings.xml

```diff
@@ -28,12 +28,12 @@
     <string name="settings_auto_restore_summary">When reinstalling an app, restore backed up settings and data.</string>
     <string name="settings_auto_restore_summary_usb">Note: Your %1$s needs to be plugged in for this to work.</string>
     <string name="settings_category_app_data_backup">App data backup</string>
-    <string name="settings_backup_apk_title">App backup</string>
+    <string name="settings_backup_apk_title">APK backup</string>
     <string name="settings_backup_apk_summary">Back up the apps themselves. Otherwise, only app data would get backed up.</string>
```

Comment:
maybe change this too?

Reply:
I feel like that description is useful, it implies "APK" is the app itself and that may be useful